portfolio/src/main/webapp/script.js

```diff
@@ -74,42 +79,47 @@ function createListElement(text) {
 }
 
 /** Deletes all comments. */
-function deleteAllComments() {
+function deleteAllComments(blogNumber) {
   console.log("Deleting all comments");
-  fetch('/delete-comment', {method: 'POST'}).then(loadCommentsSection);
+  var queryString = '/delete-comment?blog-number=' + blogNumber;
```

Comment:
var is older javascript syntax; modern (ECMAScript 5) syntax is to use const/let. Use const when a variable will not change (similar to Java's `final`), use let when a variable will be reassigned.

```suggestion
  const queryString = '/delete-comment?blog-number=' + blogNumber;
```